connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -93,13 +95,15 @@
     private static final Logger log = LoggerFactory.getLogger(DistributedHerder.class);
 
     private static final long RECONFIGURE_CONNECTOR_TASKS_BACKOFF_MS = 250;
+    private static final int BULK_THREAD_POOL_SIZE = 8;
```

Comment:
I'd err on the side of not adding configs for now and only add them if we find a need, so this seems fine.